signer/src/storage/postgres.rs

```diff
@@ -133,6 +132,35 @@ struct DepositStatusSummary {
     signers_public_key: PublicKeyXOnly,
 }
 
+/// A convenience struct for retrieving a deposit request report
+#[derive(sqlx::FromRow)]
+pub struct WithdrawalStatusSummary {
+    /// The current signer may not have a record of their vote for
+    /// the deposit. When that happens the `can_accept` and
+    /// `can_sign` fields will be None.
+    is_accepted: Option<bool>,
+    /// The height of the bitcoin chain tip during the execution of the
+    /// contract call that generated the withdrawal request.
+    #[sqlx(try_from = "i64")]
+    bitcoin_block_height: u64,
+    /// The amount associated with the deposit UTXO in sats.
+    #[sqlx(try_from = "i64")]
+    amount: u64,
+    /// The maximum amount to spend for the bitcoin miner fee when sweeping
+    /// in the funds.
+    #[sqlx(try_from = "i64")]
+    max_fee: u64,
+    /// The reclaim script for the deposit.
```

Comment:
Do withdrawals have reclaim scripts?

Reply:
Na, I just didn't update the comments. Will fix.

Reply:
Fixed in https://github.com/stacks-network/sbtc/pull/1344/commits/25dda0e8e426738c74bcd04365bcc2bcad8c5fea.